### DISCUSSION TITLE
Mean value is zero in O-RAN NearRT RIC project

### DISCUSSION THREAD

**Discussion by rajugbv** [srsran/srsRAN_Project#545] [General Help]
Hi everyone,

I am facing trouble while running the O-RAN NearRT-RIC and xApp project. As you can see, I am not getting the mean value here. I am trying to retrieve data for these metrics:

    DRB.UEThpDl - DL throughput
    DRB.UEThpUl - UL throughput
    DRB.RlcPacketDropRateDl - DL packet success rate

I have attached the gNB conf file and xApp code file. Please ask me if anything else is needed for analysis.

Below is the output of the xApp terminal:

      **1, KPM v2 ind_msg latency > 1377233325 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
      2, KPM v2 ind_msg latency > 1360456110 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
      3, KPM v2 ind_msg latency > 1343678895 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
      4, KPM v2 ind_msg latency > 1326901680 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
      5, KPM v2 ind_msg latency > 1310124465 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
      6, KPM v2 ind_msg latency > 1293347250 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
      7, KPM v2 ind_msg latency > 1276570035 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
      8, KPM v2 ind_msg latency > 1259792820 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
      9, KPM v2 ind_msg latency > 1243015605 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
     10, KPM v2 ind_msg latency > 1226238390 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
     11, KPM v2 ind_msg latency > 1209461175 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
     12, KPM v2 ind_msg latency > 1192683960 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
     13, KPM v2 ind_msg latency > 1175906745 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
     14, KPM v2 ind_msg latency > 1159129530 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
     15, KPM v2 ind_msg latency > 1142352315 s (minimum time unit is in second) from E2-node type 2 ID 411
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0
meas record INTEGER_MEAS_VALUE value 0**
[xappand gnb_conf.zip](https://github.com/srsran/srsRAN_Project/files/14876381/xappand.gnb_conf.zip)

**Comment by pgawlowicz** [comment]
Hi, do you send some traffic in DL or UL?
